docs/detections/alerts-view-details.asciidoc

```diff
@@ -17,6 +17,11 @@ The alert details flyout contains these informational tabs:
 [role="screenshot"]
 image::images/alert-details-flyout.png[Alert details flyout, 90%]
 
+Hover over fields on the *Overview* and *Table* tabs to display available <<inline-actions, inline actions>>. 
```

Comment:
```suggestion
Hover over fields on the *Overview* or *Table* tabs to display available <<inline-actions, inline actions>>. 
```

Reply:
I think `and` works here, because the user can do this on both tabs, **Overview** and **Table**, not one or the other.